repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/gatherinfocontract/GatherTestInformation.java

```diff
@@ -34,23 +35,16 @@
     private Set<FailureLocation> failureLocations;
     private Set<String> failureNames;
     private String failingModulePath;
+    private boolean skipSettingStatusInformation;
 
     private ContractForGatherTestInformation contract;
 
-    public GatherTestInformation(ProjectInspector inspector, ContractForGatherTestInformation contract) {
+    public GatherTestInformation(ProjectInspector inspector, ContractForGatherTestInformation contract, boolean skipSettingStatusInformation) {
```

Comment:
Based on the way you instantiate GatherTestInformation in the inspectors, it seems that there is a correlation between `skipSettingStatusInformation` and the contracts `BuildShouldPass` and `BuildShouldFail`. If that is true, maybe a javadoc would be nice here.